permit_jsligo/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "permit-cameligo",
+  "version": "1.0.0",
+  "description": "A modular example permit contract on Tezos written in Ligolang.",
+  "scripts": {
+    "lint": "eslint . --ext .ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "MIT",
```

you can also add bug, homepage and keywords entries, similar as in my MR https://github.com/der-alter/permit-cameligo/blob/init/package.json